pkg/runtime/ast/create_index.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ast
+
+import "strings"
+
+var (
+	_ Statement = (*CreateIndexStatement)(nil)
+	_ Restorer  = (*CreateIndexStatement)(nil)
+)
+
+type CreateIndexStatement struct {
+	IndexName string
+	Table     TableName
+	Keys      []*IndexPartSpec
+}
+
+func (c *CreateIndexStatement) CntParams() int {
+	return 0
+}
+
+func (c *CreateIndexStatement) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
+	sb.WriteString("CREATE INDEX ")
```

Comment:
`create index` has other grammar. This pr only implement the simple grammar?

https://dev.mysql.com/doc/refman/8.0/en/create-index.html

```
CREATE [UNIQUE | FULLTEXT | SPATIAL] INDEX index_name
    [index_type]
    ON tbl_name (key_part,...)
    [index_option]
    [algorithm_option | lock_option] ...
```

Reply:
> lock_option

I suggest we are as consistent as possible with MySQL.

If some grammar is difficult and not common, we are not implemented temporarily.

But some grammars are important, e.g: `UNIQUE`

Reply:
> 

ok~